### PR TITLE
Initial version intro page Structure section

### PR DESCRIPTION
**Pull request by kjellp**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [x] Documentation change
- [ ] Language translation

Drafted a new high level intro page for the Structure section, and linked it into the left menu.

### Mentions:
@norling @lilachic @tuzov

### CODE REVIEW

File: docs/structure.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+Inter-communication between services
+------------------------------------
+
+There are 3 main ways that the system is passing on information and persist state in the system, i) though AMQP messages sent from and to microservices, ii) changes in the database of the status of a file being processed in SDA and finally iii) location and state of files in either of the three file storage areas.
+
+###AMQP messaging - Rabbit MQ
+
+The orchestration of any action to be performed by the micro services, is managed through the appropriate mq message being posted at the mq broker service, from where microservices will pick up messages about work to be performed. Each microservice will thus normally only process one type of messages/jobs from a specific mq queue, and have a predefined type of next message to post once the current task is completed, for the next microservice in the pipeline to carry on the next action needed.
```
Comment (blankdots) [inline docs/structure.md:50]:
```suggestion
The orchestration of any action to be performed by the micro services, is managed through the appropriate AMQP message being posted at the RabbitMQ broker service, from where microservices will pick up messages about work to be performed. Each microservice will thus normally only process one type of messages/jobs from a specific AMQP queue, and have a predefined type of next message to post once the current task is completed, for the next microservice in the pipeline to carry on the next action needed.
```

File: docs/structure.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+To support different needs of different deployment locations, SDA is heavily configurable in several aspects. For the main archive storage, SDA supports both S3 storage and POSIX filesystem storage options, utilizing the exact same microservices with different parameters.
+
+For other storage dependent functionality, such as upload areas (aka inbox) and download areas (aka outbox), there are different choices of microservices (using different storage technology and transfer protocols) that can be orchestrated together with the main SDA microservices to meet local needs and requirements. 
+
+
+
+Inter-communication between services
+------------------------------------
+
+There are 3 main ways that the system is passing on information and persist state in the system, i) though AMQP messages sent from and to microservices, ii) changes in the database of the status of a file being processed in SDA and finally iii) location and state of files in either of the three file storage areas.
```
Comment (blankdots) [inline docs/structure.md:46]: **optional**
```suggestion
There are 3 main ways that the system is passing on information and persist state in the system:

1. though AMQP messages sent from and to micro services;
2. changes in the database of the status of a file being processed via the the `sda-pipeline`;
3. location and state of files in either of the three file storage areas.
```

**Review by blankdots — COMMENTED** [review]
idea is to be consistent in terminology e.g. `mq` vs `AMQP`

**Review by blankdots — APPROVED** [review]

File: docs/structure.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+For other storage dependent functionality, such as upload areas (aka inbox) and download areas (aka outbox), there are different choices of microservices (using different storage technology and transfer protocols) that can be orchestrated together with the main SDA microservices to meet local needs and requirements. 
+
+
+
+Inter-communication between services
+------------------------------------
+
+There are 3 main ways that the system is passing on information and persist state in the system:
+
+1. though AMQP messages sent from and to micro services;
```
Comment (pontus) [inline docs/structure.md:48]:
```suggestion
1. through AMQP messages sent from and to micro services;
```

Typo.

**Review by pontus — APPROVED** [review]

**Review by jbygdell — APPROVED** [review]

**Pull request merged**